doris/install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+wget --timestamping https://apache-doris-releases.oss-accelerate.aliyuncs.com/${DORIS_PACKAGE}.tar.gz
+mkdir ${DORIS_PACKAGE}
+tar -xvf ./${DORIS_PACKAGE}.tar.gz --strip-components 1 -C ./${DORIS_PACKAGE}
+
+echo "storage_page_cache_limit=60%" >> ./${DORIS_PACKAGE}/be/conf/be.conf
```

Comment:
We should rather use for all benchmarked databases the default settings, i.e. not modify configuration.

If we really need to do that, then please add a comment that explains why the default doesn't work.

Reply:
removed